modules/end-user-guide/partials/proc_overriding-devfile-values-using-factory-parameters.adoc

```diff
@@ -135,7 +137,7 @@ projects:
 ...
 ----
 
-When overriding attributes, everything that follows the `attributes` keyword is interpreted as an attribute name, so a user can use dot-separated names:
+When overriding attributes, everything that follows the `attributes` keyword is interpreted as an attribute name. The user can use dot-separated names:
```

Comment:
Can you address the user directly? IBM SG states:
```
Second person
As much as possible, use the second person, which is indicated by the personal pronoun you. The second person focuses on the reader.
> When you create a database, you must provide a unique name.

To instruct the user to perform an action, use the imperative mood. The subject of an imperative sentence is understood to be you.
> To begin processing, press Enter.
```

In other words, if you are addressing the user, avoid using "the user." 

On the other hand, if you are not talking about "the user" to someone else, such as an administrator, then keep it and consider being more specific about the user role. For example, if you're talking to an administrator about a developers, you might say something like "The developer can ...."